crates/modelardb_server/src/parser.rs

```diff
@@ -176,6 +176,20 @@ impl ModelarDbDialect {
         }
     }
 
+    /// Return its value as a [`f32`] if the next [`Token`] is a
+    /// [`Token::Number`], otherwise a [`ParserError`] is returned.
+    fn parse_positive_literal_f32(&self, parser: &mut Parser) -> Result<f32, ParserError> {
+        let token_with_location = parser.next_token();
+        match token_with_location.token {
+            Token::Number(maybe_f32, _) => maybe_f32.parse::<f32>().map_err(|error| {
```

Comment:
Is this locale-dependent, i.e., 0.5 for en_US, but 0,5 for da_DK?

Reply:
Based on the documentation it seems only `en_US` is supported as `f32`s can only be parsed if they follow this [grammar](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.from_str).

Reply:
OK, good.